src/components/landing-layout/footer/social-links.tsx

```diff
@@ -16,11 +16,11 @@ export const SocialLink: React.FC = () => {
       <Trans
         i18nKey='landing.footer.followUs'
         components={[
-          <ExternalLink href={links.githubOrg} icon='github' text='GitHub' />,
-          <ExternalLink href={links.community} icon='users' text='Discourse' />,
-          <ExternalLink href={links.chat} icon='comment' text='Matrix' />,
-          <ExternalLink href={links.mastodon} icon='mastodon' text='Mastodon' />,
-          <ExternalLink href={links.translate} icon='globe' text='POEditor' />
+          <ExternalLink href={links.githubOrg} icon='github' key={'github'} text='GitHub' />,
+          <ExternalLink href={links.community} icon='users' key={'users'} text='Discourse' />,
+          <ExternalLink href={links.chat} icon='comment' key={'comment'} text='Matrix' />,
+          <ExternalLink href={links.mastodon} icon='mastodon' key={'mastodon'} text='Mastodon' />,
+          <ExternalLink href={links.translate} icon='globe' key={'globe'} text='POEditor' />
```

Comment:
The keys here seem to be wrong. These seem better to me:
```suggestion
          <ExternalLink href={links.githubOrg} icon='github' key={'github'} text='GitHub' />,
          <ExternalLink href={links.community} icon='users' key={'discourse'} text='Discourse' />,
          <ExternalLink href={links.chat} icon='comment' key={'matrix'} text='Matrix' />,
          <ExternalLink href={links.mastodon} icon='mastodon' key={'mastodon'} text='Mastodon' />,
          <ExternalLink href={links.translate} icon='globe' key={'poeditor'} text='POEditor' />
```

Reply:
They aren't wrong. They just have to be unique. I could also use numbers.

Reply:
Why do we need to add key's here anyway?

Reply:
Because the linter says so. It actually makes sense because it's an array and the order doesn't have to be fixed. We could rework this into static components.

Reply:
I think we can keep them like this.